app/recordtransfer/jobs.py

```diff
@@ -9,7 +9,7 @@
 from django.core.files.base import ContentFile
 from django.utils import timezone
 
-from recordtransfer.models import Job, Submission, User
+from recordtransfer.models import Job, Submission, UploadSession, User
 from recordtransfer.utils import zip_directory
 
 LOGGER = logging.getLogger("rq.worker")
```

Comment:
A special logger is not needed here either.

```suggestion
LOGGER = logging.getLogger(__name__)
```